### PR TITLE
Rust and Go updates (ws-images)

### PR DESCRIPTION
**Pull request by filiptronicek**
## Description

This PR adds a changelog entry to let users know about changes done in https://github.com/gitpod-io/workspace-images/pull/1094 and https://github.com/gitpod-io/workspace-images/pull/1041.

Todo:
- [x] add a nice image

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3733"></a>

### CODE REVIEW

File: src/lib/contents/changelog/2023-06-11-go-rust-updates.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Workspace images: Go 1.20.5 & Rust 1.70"
+alt: screenshot of the new workspace creation page
+date: 2023-06-11
```
Comment (Siddhant-K-code) [inline src/lib/contents/changelog/2023-06-11-go-rust-updates.md:4]: Did you mean 11 July?

```suggestion
date: 2023-07-11
```

Comment (filiptronicek) [reply src/lib/contents/changelog/2023-06-11-go-rust-updates.md:4]: Nope... it just takes a long time to get this published 😆

Comment (Siddhant-K-code) [reply src/lib/contents/changelog/2023-06-11-go-rust-updates.md:4]: But, if we keep it June, it wouldn't be the latest changelog. As we shipped [this changelog](https://www.gitpod.io/changelog/https-workspace-ports) on June 19.

**Comment by filiptronicek** [comment]
No longer needed, we decided to take care of these in workflows-images' changelog.

**Pull request closed without merging**